docroot/sites/uipress.uiowa.edu/modules/uipress_core/templates/block--cartbuttons-block.html.twig

```diff
@@ -0,0 +1,14 @@
+{% if content['#href'] != '' %}
+  <div class="layout-builder-block">
```

Comment:
Why are we wrapping this div around this?

Reply:
Not sure I follow. Do they not need to be in a div? Or is the div something that should be added in php instead of twig?

Reply:
They may need to be in a `<div>`. I'm not sure if the templating inheritance results in them be wrapped already or not. I guess the thing I was asking about is the use of the `.layout-builder-block` class.